### PR TITLE
Update instructions to install Superdesk/Liveblog

### PR DESCRIPTION
**Pull request by naspeh**
It is better to look into these links:
- https://github.com/superdesk/fireq/tree/docs/files/superdesk-dev it should be pretty usable :)
- https://github.com/superdesk/fireq/tree/docs/files/superdesk needs more clarifications yet

I'm going to share `superdesk-dev` first in `#superdesk` channel.

Cc: @petrjasek, @hlmnrmr, @jerome-poisson

### CODE REVIEW

File: files/superdesk-dev/deploy [hunk 1]
```diff
@@ -290,7 +290,7 @@ fi
 [ -d /var/log/superdesk ] || mkdir -p /var/log/superdesk
 cat <<"EOF" > /etc/supervisor/conf.d/superdesk.conf
 [program:rest]
-command=/bin/sh -c '. /opt/superdesk/env/bin/activate && exec gunicorn -b 0.0.0.0:5000 -t 300 wsgi'
+command=/bin/sh -c '. /opt/superdesk/env/bin/activate && exec gunicorn -b 0.0.0.0:5000 -t 300 wsgi --reload'
```
Comment (petrjasek) [inline files/superdesk-dev/deploy]: for development would be useful to also reload workers on change

File: files/superdesk/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+**Minimal requirements:**
+2GB RAM, 4GB Free space
+
+## Install a superdesk on fresh Ubuntu 16.04
```
Comment (petrjasek) [inline files/superdesk/README.md]: what about update?

File: files/superdesk-dev/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+###[Prepare LXC](../../docs/lxc.md)
+
```
Comment (petrjasek) [inline files/superdesk-dev/README.md]: imo it can use some info on what systems you can actually run this.

Comment (petrjasek) [inline files/superdesk-dev/README.md]: now I see that in this lxc part, imo we would need eventually something you can run from mac or windows, maybe we could prepare a vagrant file for that

File: files/superdesk-dev/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+# - /var/cache/fireq for pip, npm, dpkg caches and logs
+(echo name=$lxc; curl https://raw.githubusercontent.com/superdesk/fireq/master/files/superdesk-dev/lxc-init) | sudo bash
+(echo host=$lxc; curl https://raw.githubusercontent.com/superdesk/fireq/master/files/superdesk-dev/install) | ssh root@$lxc
+
+# open http://$lxc in browser to access superdesk
+
+# current directory is mounted inside the container,
+# some files could be created by root during installation, so
+sudo chown -R <your_user> .
+ll ./client-core # superdesk-client-core
```
Comment (petrjasek) [inline files/superdesk-dev/README.md]: imo we shouldn't use `ll` here, not everyone will have this alias configured

File: files/superdesk-dev/README.md (hunk 3 above)
Comment (naspeh) [inline files/superdesk-dev/README.md]: yep, this is for linux systems
yep, probably a vagrant file for mac or windows, but later (maybe someone could help with this)

File: files/superdesk/README.md (hunk 2 above)
Comment (naspeh) [inline files/superdesk/README.md]: I'll add this (it's user installations, needs more info yet)

something like
```
cd /opt/superdesk
git pull
# and run install script again, it's safe to run it many times
curl https://raw.githubusercontent.com/superdesk/fireq/master/files/superdesk/install | sudo bash
```

File: files/superdesk-dev/deploy (hunk 1 above)
Comment (naspeh) [inline files/superdesk-dev/deploy]: yep, it probably needs a separate watcher and this watcher will restart `wamp`, `work`, `beat` (I'll add it later)

**Comment by naspeh** [comment]
Ok, I'm merging it to share in `#superdesk` channel, other updates will be in new PR

**Pull request merged**